ansible/roles/common/tasks/dependencies.yml

```diff
@@ -36,20 +35,48 @@
   apt:
     update_cache: no
     state: present
-    name: libgfortran-5-dev
-  when: raspbian_version.stdout == 'stretch'
+    name:
+      - libgfortran-5-dev
+      - predict
+  when: ansible_distribution_release == 'stretch'
 
 - name: install dependencies (non-stretch)
   become: yes
   apt:
     update_cache: no
     state: present
     name: libgfortran5
-  when: raspbian_version.stdout != 'stretch'
+  when: ansible_distribution_release != 'stretch'
+
+- name: install wkhtmltox (wkhtmltoimg) arm
+  become: yes
+  apt:
+    deb: "https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.raspberrypi.{{ ansible_distribution_release }}_armhf.deb"
+    state: present
+  when: ansible_architecture == 'armv7l'
+
+- name: install wkhtmltox (wkhtmltoimg) x86_64
+  become: yes
+  apt:
+    deb: "https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.{{ ansible_distribution_release }}_amd64.deb"
+    state: present
+  when: ansible_architecture == 'x86_64'
+
+# predict is not packaged anymore after stretch
```

Comment:
:frowning_face:

Reply:
for predict ? the debian package seems to have a bunch of patches to it and is even older than latest from website. I don't really know what would be the best plan, maybe using pypredict in a script

Reply:
Yeah, it's a shame the debian package became unmaintained.

What you have done here is probably fine in the short term, we should perhaps open a new issue discussing other options though?

Reply:
absolutely, in my hackish pile of python it did use pypredict, I haven't looked here how and when is predict used, but since there is already python it could be added with the rest.

Reply:
:+1: Let's move this here: https://github.com/jekhokie/raspberry-noaa-v2/issues/462 it should happen irrespective of this PR :smile_cat: